pkgs/development/python-modules/qscintilla-qt5/default.nix

```diff
@@ -27,6 +27,8 @@ buildPythonPackage {
     lndir ${pyqt5} $out
     rm -rf "$out/nix-support"
     cd Python
+    sed -i configure.py \
```

Comment:
`sed` is usually not the most future proof option, since if the upstream file changes, this will fail to apply silently.

`substituteInPlace` should be preferred for this reason (https://nixos.org/nixpkgs/manual/#fun-substituteInPlace), or a plain old patch if you prefer.

Reply:
OK, I used `substituteInPlace` instead.